ibrahim/styles.scss

```diff
@@ -0,0 +1,265 @@
+* {
+  box-sizing: border-box;
+}
+
+/* Undoing defaults */
+body,
+.body {
+  transition: width 750ms, margin 750ms;
+}
+
+body {
+  margin: 0;
+}
+
+ul {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+figure {
+  margin: 0;
+}
+
+img {
+  height: auto;
+  width: 100%;
+}
+
+/* ---------- */
+.body {
+  margin: 0;
+}
+
+body {
+  font-family: "Quicksand", sans-serif;
+  color: #1e1e1e;
+  font-size: 10px;
+  background: linear-gradient(#f0f0f0, #ffffff);
+}
+
+.text {
+  margin: 20px 25px;
+}
+
+h1 {
+  color: #015759;
+  font-family: "Comfortaa", sans-serif;
+  font-size: 36px;
+}
+
+h2 {
+  color: #015759;
+  font-family: "Comfortaa", sans-serif;
+  font-size: 18px;
+}
+
+/* Navigation */
+nav {
+  background-color: #1e1e1e;
+  text-align: center;
+
+  li {
+    display: inline-block;
+    margin-left: -4px;
+  }
+
+  li:first-child {
+    margin-left: 0;
+  }
+
+  a {
+    display: block;
+    background-color: #1e1e1e;
+    color: white;
+    padding: 14px 26px;
+    text-decoration: none;
+    font-family: "Raleway", sans-serif;
+    font-size: 14px;
+    font-weight: 700;
+    transition: all 250ms;
+  }
+
+  a:hover,
+  a:focus {
```

Comment:
See previous note -- you can also use the `&` parent reference to write pseudoclasses 🙌